### ISSUE TITLE
Missing feedbackErrorMessage when submitting incomplete feedback form

### ISSUE THREAD

**Issue by pdehaan** [mozilla/testpilot#531]
Found in http://testpilot-dev.us-east-1.elasticbeanstalk.com/__version__ (9bf2838358cfb7001ed54122406e9c9b9968e0c9)

``` json
{"version": "", 
  "source": "https://github.com/mozilla/testpilot", 
  "commit": "9bf2838358cfb7001ed54122406e9c9b9968e0c9"}
```

---
### Steps to reproduce:
1. Go to http://testpilot-dev.us-east-1.elasticbeanstalk.com/experiments/universal-search and enable the add-on.
2. Use it. Love it.
3. Disable it.
4. In the "Why don't you love me?" modal pop-up, click the "Submit Feedback" button without offering any feedback.
### Actual results:

> feedbackErrorMessage

![firefox_test_pilot_-_universal_search](https://cloud.githubusercontent.com/assets/557895/14027282/17d40a36-f1b4-11e5-84ca-8a773d3318be.png)
### Expected results:

An actionable error message. I don't see any "feedbackErrorMessage" key in the [/locales/en-US/app.l20n](https://github.com/mozilla/testpilot/blob/master/locales/en-US/app.l20n) file, so I think we're just missing the key/value.

**Comment by pdehaan** [comment]
On a more personal filled note, if you _force_ me to supply feedback upon cancelling something, you're likely to get garbage feedback/selections and "qwerty" text, or some very hurtful and profanity-laced comments. We may want to consider the feedback as optional and adding a "Skip" or "No thanks" link above the "Cancel" link.

**Comment by pdehaan** [comment]
Fixed via #535. Closing.
